frontend/lib/ui/artefact_page/test_event_log_expandable.dart

```diff
@@ -16,66 +14,57 @@ class TestEventLogExpandable extends ConsumerWidget {
   final bool initiallyExpanded;
 
   @override
-  Widget build(BuildContext context, WidgetRef ref) {
-    final testEvents = ref.watch(testEventsProvider(testExecutionId));
-
-    return Expandable(
-      title: const Text('Event Log'),
-      initiallyExpanded: initiallyExpanded,
-      children: <Widget>[
-        testEvents.when(
```

Comment:
Aren't we deleting the Expandable here, I don't see it anywhere in the new code

Reply:
Great catch

Reply:
I tried manually testing everything now, so hopefully I didn't miss anything else